Calculator/Calculator/CalculatorLogic.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Calculator
+{
+    class Operation
```

Comment:
It's a good practice to name class and file with the same name. So it's better to rename CalculatorLogic.cs file to Operation.